lib/operators/with-status.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { expect } from "chai";
+import { filter, map } from "rxjs/operators";
+import { TestScheduler } from "rxjs/testing";
+
+import { withStatus } from "./with-status";
+import { EventStatus, EventWithStatus } from "../with-status";
+
+const comparator = (actual: unknown, expected: unknown) => {
+  expect(actual).to.deep.equal(expected);
+};
+
+describe("withStatus operator", () => {
```

Comment:
as you `withStatus` function in the operator, I don't know if it's wort to duplicate tests